test/bn_vectors.js

```diff
@@ -76,6 +76,9 @@ sjcl.test.vector.bn_powermod = [
   // // This was disabled because it's slow (~2.5s with `rhino -O 9`). Once
   // // a better powermod algorithm is implemented (e.g., using Montgomery
   // // reduction), this can be re-enabled.
+  //
+  // // Note:  This example can't be done with Montgomery reduction anyways
```

Comment:
You are totally right (I was too lazy to read the whole file sorry). It would still be awesome to re-enable this test.